lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -265,14 +265,23 @@ def append_size(self, position, size):
                                position=position)
 
     def add_auto_adjustable_area(self, use_axes, pad=0.1, adjust_dirs=None):
+        """
+        Add auto-adjustable padding around axes to take their decorations
```

Comment:
```suggestion
        Add auto-adjustable padding around ``Axes``es to take their decorations
```
??? Would like to have the capitalized `Axes`, making that plural is hard.

Alternatively: 
```suggestion
        Add auto-adjustable padding around *use_axes* to take their decorations
```